### ISSUE TITLE
Error when installing 

### ISSUE THREAD

**Issue by Shinoby1992** [stereodruid/OSMOSIS#4]
15:35:46 T:123145494646784  NOTICE: [plugin.video.osmosis-0.0.1]: get_params
15:35:46 T:123145494646784  NOTICE: [plugin.video.osmosis-0.0.1]: Mode: None
15:35:46 T:123145494646784  NOTICE: [plugin.video.osmosis-0.0.1]: Name: None
15:35:46 T:123145494646784  NOTICE: [plugin.video.osmosis-0.0.1]: getSources
15:35:46 T:123145494646784  NOTICE: Previous line repeats 1 times.
15:35:46 T:123145494646784  NOTICE: [plugin.video.osmosis-0.0.1]: addDir
15:35:46 T:123145494646784  NOTICE: Previous line repeats 3 times.
15:35:46 T:123145494646784   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: object of type 'NoneType' has no len()
                                            Traceback (most recent call last):
                                              File "/Users/abc/Library/Application Support/Kodi/addons/plugin.video.osmosis/service.py", line 73, in <module>
                                                strm_update()
                                              File "/Users/abc/Library/Application Support/Kodi/addons/plugin.video.osmosis/service.py", line 66, in strm_update
                                                for i in range(len(thelist)):
                                            TypeError: object of type 'NoneType' has no len()
                                            -->End of Python script error report<--
15:35:46 T:123145494646784    INFO: Python script stopped
15:35:46 T:123145494646784   DEBUG: Thread LanguageInvoker 123145494646784 terminating

**Comment by stereodruid** [comment]
Etwas mehr infos wären toll. Systen, Kodi version.  Wurde das Addon installiert?

**Comment by Shinoby1992** [comment]
Kodi 16.1 Stable
Mac OSX 10.12 (Sierra)

Direkt nach installation kommt im Kodi Fehler, und debug log ist oben

**Comment by stereodruid** [comment]
Es scheint so als versuche das auto update script eib update durchzuführen. Obwohl es noch keinen Content gibt.

**Comment by Shinoby1992** [comment]
Ich habe im Kodi xstream als gadget kann das die Ursache sein?

**Comment by stereodruid** [comment]
Eigentlich nicht. Ist osmosis installiert?

**Comment by stereodruid** [comment]
@Shinoby1992  
Welchen skin verwendest du?

**Comment by Shinoby1992** [comment]
Ja ist installiert.
Aen Nox 5 Silvo

ich hab gerade ausprobiert über osmosis auf Kinox (xstream) und hab einen film gestartet. 
Müsste ich da nicht infos bekommen über den film oder was genau macht das plugin? ich dachte die werden über IMDB gescraped?

**Comment by stereodruid** [comment]
Einfach mal ne serie bei  unter kinox hinzufüge dan kodi neu starten und danach berichten. Danke.

**Comment by stereodruid** [comment]
Das addon fügt filme und serien zu deiner datenbank hinzu. Wenn du zum Beispiel auf kinox#filme#kinofilme ,also auf den  Menüpunkt "Kinofilme" das Kontextmenü offnest, steht da create strm, das auswählen und im nächsten TV auswählen, Fertig. Für Serien Episodes Kinox nutzen. Danach noch deinem Kodi sagen wo die Filme/Serien liegen. Das automatische aktualisieren bedeutet das die Kinofilme in deiner Library immer aktuell bleiben und die neusten hinzugefügt werden. Das gleiche gilt für serien.

**Comment by stereodruid** [comment]
Für das Problem nach der Installation ,habe ich eine Lösung gefunden. Werde sie bald implementieren.

**Comment by stereodruid** [comment]
@Shinoby1992 
Theoretisch brauchst du, wenn du alles eingerichtet hast, xstream nicht stendig öffnen um zu sehen ob neue Filme erschienen sind oder Setien. Die Filme und Serien weden, automatisch  deiner Bibliothek als Streamreferenz hinzugefügt, mit allen dazugehörigen infos von IMDB oder einem anderen scraper.

**Comment by Shinoby1992** [comment]
okay nice ich teste es gleich mal aus.
Hat das ein bestimmten Vorteil im vergleich zu der Widget Nutzung?
 , sag bescheid sobald der fehler behoben ist

**Comment by stereodruid** [comment]
Ja, die Library sortiert deine Inhalte und du hast mehr Infos zu den Filmen/Serien.

**Comment by stereodruid** [comment]
Mit Library meine ich die KODI interne Datenbank für Filme und Serien.
